mep/common/tests.py

```diff
@@ -332,6 +334,110 @@ def test_dict_item(self):
         # integer value
         assert dict_item({13: 7}, 13) is 7
 
+class TestCheckboxFieldset(TestCase):
```

Comment:
I glanced at the other code bases where we used the facet fields stuff and it looks like we never wrote tests for them - thanks for adding these.